pyexif/__init__.py

```diff
@@ -66,12 +66,14 @@ def _runproc(cmd, fpath=None):
 
 
 class ExifEditor(object):
-    def __init__(self, photo=None, save_backup=False):
+    def __init__(self, photo=None, save_backup=False, extra_opt=None):
         self.save_backup = save_backup
         if not save_backup:
             self._optExpr = "-overwrite_original_in_place"
         else:
             self._optExpr = ""
+        if extra_opt:
+            self._optExpr += " " + extra_opt
```

Comment:
Minor nit: I think it's cleaner to write:

`self._optExpr += " %s" % extra_opt